README.md

```diff
@@ -6,239 +6,46 @@
+
 Serializers describe _which_ attributes and relationships should be serialized.
+
 Adapters describe _how_ attributes and relationships should be serialized.
 
-By default AMS will use the **Attributes Adapter**. But we strongly advise you to use **JsonApi Adapter** that follows 1.0 of the format specified in [jsonapi.org/format](http://jsonapi.org/format).
-Check how to change the adapter in the sections bellow.
+SerializableResource co-ordinates the resource, Adapter and Serializer to produce the
+resource serialization, which has the `#as_json`, `#to_json` and `#serializable_hash`
+methods used by the Rails JSON Renderer. (SerializableResource actually delegates
+these methods to the adapter.)
+
+By default ActiveModelSerializers will use the **Attributes Adapter**.
+But we strongly advise you to use **JsonApi Adapter**, which
+follows 1.0 of the format specified in [jsonapi.org/format](http://jsonapi.org/format).
+Check how to change the adapter in the sections below.
 
 # Documentation
 
 Master
 
-- [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers)
-- [Guides](https://github.com/rails-api/active_model_serializers/tree/master/docs)
-
-# RELEASE CANDIDATE, PLEASE READ
-
-This is the master branch of AMS. It will become the `0.10.0` release when it's
-ready. Currently this is a release candidate. This is **not** backward
-compatible with `0.9.0` or `0.8.0`.
-
-`0.10.x` will be based on the `0.8.0` code, but with a more flexible
-architecture. We'd love your help. [Learn how you can help here.](https://github.com/rails-api/active_model_serializers/blob/master/CONTRIBUTING.md)
-
-## Example
-
-Given two models, a `Post(title: string, body: text)` and a
-`Comment(name: string, body: text, post_id: integer)`, you will have two
-serializers:
-
-```ruby
-class PostSerializer < ActiveModel::Serializer
-  cache key: 'posts', expires_in: 3.hours
-  attributes :title, :body
-
-  has_many :comments
-end
-```
-
-and
-
-```ruby
-class CommentSerializer < ActiveModel::Serializer
-  attributes :name, :body
-
-  belongs_to :post
-end
-```
-
-Generally speaking, you as a user of AMS will write (or generate) these
-serializer classes. If you want to use a different adapter, such as a JsonApi, you can
-change this in an initializer:
-
-```ruby
-ActiveModel::Serializer.config.adapter = ActiveModel::Serializer::Adapter::JsonApi
-```
-
-or
+- [0.10 Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
+  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers)
+  - [Guides](https://github.com/rails-api/active_model_serializers/tree/master/docs)
+- [0.9 Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
+- [0.8 Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
```

Comment:
Older docs in 0-8 and 0-9 are pretty good. We should bring some of that back.

- https://github.com/rails-api/active_model_serializers/tree/0-8-stable
- https://github.com/rails-api/active_model_serializers/tree/0-8-stable